problems/loops.js

```diff
@@ -143,16 +228,33 @@ function fizzBuzzPart1() {}
  * @returns {Array}
 
  */
-function fizzBuzzPart2() {}
-
+const fizzBuzzPart2 = () => {
+  let newArray = [];
+  for (let i = 1; i <= 100; i++) {
+    if (i % 3 === 0) {
+      newArray.push("Fizz");
+    } else if (i % 5 === 0) {
+      newArray.push("Buzz");
+    } else if (i % 3 === 0 && i % 5 === 0) {
```

Comment:
Remember only one condition can satisfy the if else if else blocks, so order matters. Rethink this order.